lib/fastlane/plugin/firebase_app_distribution/actions/firebase_app_distribution_login.rb

```diff
@@ -16,20 +16,19 @@ def self.run(params)
         authorizer = Google::Auth::UserAuthorizer.new(client_id, SCOPE, nil)
         url = authorizer.get_authorization_url(base_url: OOB_URI)
 
-        UI.message("Please open the following address in your browser:")
+        UI.message("Open the following address in your browser and sign in with the Google account on your Firebase project:")
```

Comment:
I get what your trying to convey here, but the part that says “on your Firebase project” sounds weird. Consider removing it. I think it’s _okay_ to assume that the google account they use will be the same one that has access to their firebase project.